src/vm-repair/azext_vm_repair/_help.py

```diff
@@ -0,0 +1,37 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.help_files import helps
+
+helps['vm repair'] = """
+    type: group
+    short-summary: Auto repair commands to fix faulty VMs. Recommended to use all commands with the --verbose flag.
+    long-summary: >
+        VM repair commands will create a new rescue VM to fix the OS disk of the faulty VM. This is called the 'disk-swap' process. The OS disk of the faulty VM will be copied then attached on the rescue VM's data disk. From the rescue VM, one can diagnose and fix the attached data disk.
+"""
+
+helps['vm repair swap-disk'] = """
+    type: command
+    short-summary: Create a new rescue VM, and attach a copy of the OS disk from the target VM to the rescue VM as a data disk. Recommended to use with the --verbose flag.
```

Comment:
It would be good to put the recommendation in the long-summary.

 "It is recommended to use `--verbose` ..."

Reply:
The short-summary is typically a much shorter sentence, try to make your short-summaries shorter.
See [here](https://github.com/Azure/azure-cli/blob/dev/src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_help.py#L298) for an example..

So something like the following:
```suggestion
    short-summary: Swap a copy of the target VM's disk onto a rescue VM
    long-summary: Create a new rescue VM, and attach a copy of the OS disk from the target VM to the rescue VM as a data disk. Recommended to use with the --verbose flag.
```
You might want to explain that a new vm is created if an existing rescue vm is not specified.

help reference: https://github.com/Azure/azure-cli/blob/dev/doc/authoring_help.md#tips-to-write-effective-help-for-your-command